forge-gui/res/cardsfolder/upcoming/roaming_throne.txt

```diff
@@ -0,0 +1,10 @@
+Name:Roaming Throne
+ManaCost:4
+Types:Artifact Creature Golem
+PT:4/4
+K:Ward:2
+K:ETBReplacement:Other:ChooseCT
+SVar:ChooseCT:DB$ ChooseType | Type$ Creature | AILogic$ MostProminentInComputerDeck | SpellDescription$ As CARDNAME enters the battlefield, choose a creature type.
+S:Mode$ Continuous | Affected$ Card.Self | AddType$ ChosenType | Description$ CARDNAME is the chosen type in addition to its other types.
+S:Mode$ Panharmonicon | ValidCard$ Creature.Other+YouCtrl+ChosenType | Description$ If a triggered ability of another creature you control of the chosen type triggers, it triggers an additional time.
```

Comment:
It says "creature you control", so it needs inZoneBattlefield check

Reply:
should be handled?
![image](https://github.com/Card-Forge/forge/assets/8506892/a42765ab-3838-414f-9ef9-5425183e9fd7)